conf/ocsci/aws_i3_local_storage.yaml

```diff
@@ -0,0 +1,8 @@
+---
+DEPLOYMENT:
+  local_storage: true
+
+ENV_DATA:
+  platform: 'aws'
+  worker_instance_type: 'i3en.2xlarge'
+  device_size: 2300  # Value pulled from here: https://bugzilla.redhat.com/show_bug.cgi?id=1812557#c27
```

Comment:
according to https://aws.amazon.com/ec2/instance-types/, i3en.2xlarge instances has 2 x 2,500 NVMe SSD

Reply:
Yes, they do. I wasn't sure if we weren't using the entire device or some reason so I went with what was suggested. We can change this to 2500 if there isn't a good reason to have it at 2300.